README.md

```diff
@@ -2,9 +2,9 @@
 
 ## Prerequisites
 *  [Docker Desktop](https://www.docker.com/products/docker-desktop/) installed and running
-*  **All** funding-service-design apps (listed as `context` keys in [docker-compose.yml](docker-compose.yml) must be cloned and checked out in the parent directory of this repository
 
 ## How to run
+* Run `./scripts/bootstrap.sh` to clone all required repositories.
```

Comment:
does this clone all repo's to current dir, or parent dir (as per the root of this repo? Should we clarify in the readme (as per the removed comment that explicitly states where to manually clone them).

Reply:
(that said I am mildly in favour of having them cloned instead to eg some `app` directory in this folder, so that everything's contained nicely ... but not bothered enough to suggest that change here)

Reply:
Added in ca5605a